docs/source/platform/operation-registry.md

```diff
@@ -168,6 +165,23 @@ const server = new ApolloServer({
 });
 ```
 
+####5.1 Optionally, set the schema tag
+
+The plugin will automatically pickup the value of the environment variable `ENGINE_SCHEMA_TAG` and use that as the schema tag.  
+
+To override this behaviour, configure the `schemaTag` field.  
+
+```js
+const server = new ApolloServer({
+  // Existing configuration
+  plugins: [
+    require("apollo-server-plugin-operation-registry")({
+      schemaTag: 'overrideTag' // highlight-line
+    })
+  ]
+});
+```
```

Comment:
We should also mention the need to set the tag in the client push command here in order to get the operations into the manifest the plugin pulls